link-grammar/disjunct-utils.c

```diff
@@ -1063,6 +1063,9 @@ void share_disjunct_jets(Sentence sent, bool rebuild)
 		{
 			jet_table_size[dir] = js->entries[dir] + 1;
 			memset(jet_table[dir], 0, jet_table_size[dir] * sizeof(Connector *));
+
+			for (size_t i = 0; i < jet_table_size[dir]; i++)
+				jet_table[dir][i] = NULL;
```

Comment:
Didn't the memset above already accomplish the same thing?

Reply:
Yes. I removed this commit and force-pushed.